src/EdFi.AnalyticsMiddleTier.Tests/Classes/FinancialAidFact.cs

```diff
@@ -1,4 +1,9 @@
-﻿using System;
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+

```

Comment:
Here's a cool feature of the new tooling: it found a missing header and inserted it for us 😄 .